docs/conf.py

```diff
@@ -22,6 +22,13 @@
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
+import os
+import sys
+import pkg_resources
+
+
+sys.path.append(os.path.abspath('../src'))
+rqmt = pkg_resources.require('zope.container')[0]
```

Comment:
This commit contains changes not mentioned in the commit message (changes I think I've seen before in another PR)?

Reply:
I considered that part of "Also other minor doc improvements."

Reply:
Ah.  I got mixed up with the PRs and assumed this was some kind of a git rebase accident.